### ISSUE TITLE
Docker Stack Missing

### ISSUE THREAD

**Issue by mwaeckerlin** [AgustinCB/docker-api#54]
Is this project still supported? It seems that the new Docker Swarm capabilities are missing, i.e. commands such as `docker stack ls` are not implemented. Is this planned?

Currently, I use `child_process` to execute docker commands in nodejs, but your approach using the API is more elegant, so I'd like to switch to your library. This is my project wher eI would like to use your library: [mwaeckerlin/docker.js](https://github.com/mwaeckerlin/docker.js)

**Comment by AgustinCB** [comment]
I'd like to still support it, but the truth it's that I have other priorities and this project grew much more than I expected. It started as something I did in my free time to learn how to develop libraries in node and somehow now it's being used. Which is great, I'm very grateful with the community that embrace it. But it also puts a responsibility in me I'm not I can handle.

I can't promise you I'll add support to that (or any future docker feature). I'll, though, do a better job at keeping up with contributions from other people and making sure that whatever people adds is added in master. So if you want, you can always add the support yourself and contribute it back (it should be pretty straight forward).

**Comment by mwaeckerlin** [comment]
Yes, I have seen, that it is quite huge. If it were simpler, I had tried to add it, but I'd like more to focus on my part (docker (swarm) web ui), so I can't spend too much effort on your API-library. So you're the only developer?

**Comment by AgustinCB** [comment]
There're contributions from other developers, but no regular contributors.
